internal/manifest/manifest.go

```diff
@@ -38,37 +38,51 @@ type SNPReferenceValues struct {
 
 // SNPTCB represents a set of SNP TCB values.
 type SNPTCB struct {
-	BootloaderVersion SVN
-	TEEVersion        SVN
-	SNPVersion        SVN
-	MicrocodeVersion  SVN
+	BootloaderVersion NullSVN
+	TEEVersion        NullSVN
+	SNPVersion        NullSVN
+	MicrocodeVersion  NullSVN
```

Comment:
How is this different from using `*SVN`?

Reply:
I guess I was overthinking it. I thought we can avoid nil checks on the `UInt8` function this way, but I suppose we can just use a pointer since otherwise you would need to check if the value is valid before calling `UInt8`, as we are doing it now.